extension/intents/window/window.js

```diff
@@ -44,3 +44,13 @@ intentRunner.registerIntent({
     await browser.experiments.voice.openDownloads();
   },
 });
+
+intentRunner.registerIntent({
+  name: "window.combine",
+  async run(context) {
+    const currentWindow = await browser.windows.getCurrent();
+    const tabs = await browser.tabs.query({});
+    const tabsIds = tabs.map(tabInfo => tabInfo.id);
+    await browser.tabs.move(tabsIds, { windowId: currentWindow.id, index: -1 });
```

Comment:
I think this will also move tabs that are already in the current window, and might mix up the order of those tabs. So this should filter out tabs already in the current window.

Reply:
Oh, yes, indeed. I added now filtering.